env-setup.bat

```diff
@@ -0,0 +1,9 @@
+ldc2-1.35.0-windows-x64\bin\dub.exe fetch dlib
+ldc2-1.35.0-windows-x64\bin\dub.exe fetch dlib:math
+ldc2-1.35.0-windows-x64\bin\dub.exe fetch dlib:math:portable
```

Comment:
Why are those `dub fetch` commands needed/helpful?

Reply:
they're not. i just finished with my own review :) the file has been deleted to avoid confusion.
resolving.